### ISSUE TITLE
[Off-chain actions] Handle sponsor actions when DAO sponsor tx is incomplete

### ISSUE THREAD

**Issue by jtrein** [openlawteam/tribute-ui#91]
**Overview of sponsoring + off-chain voting**

When we submit a Snapshot Hub proposal the voting period starts straight away (intended). Then we need to subsequently submit a DAO sponsorship tx and the voting will start at the tx block timestamp.

_Note: There will always be some voting start time differences between the 2 systems. We can handle this explanation in the UI (#92 ). For example, in the contract when determining if the result is ready to submit (e.g. `vote.startingTime + votingPeriod <= block.timestamp` + some other shares logic in the contract) the Snapshot Hub vote may have ended but the contract is not ready for the result to be submitted._

**Issue to solve**

Basically, we need to remind the user a step is not complete. If the user does not submit the DAO sponsorship tx, the off-chain voting will not show, and the voting time will have started Today, the DAO sponsor tx call cannot be picked up where it left off. 

**Ideal solution**

We should be able to:

1) Remind the user "Off-chain voting has started!" and that they need to submit the sponsorship tx to the DAO.
2) User should be able to click a button and submit the tx only (not repeat the entire sponsorship process over).

This solution is most likely the easiest to implement, and it encourages users to finish their wallet transactions, not ignore them.
